components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -90,6 +91,278 @@ def test_mkdw002_dcclink(dash_dcc):
+            },
+            "height": 500,
+            "width": 800,
+            "margin": {"r": 250},
+            "title": {
+                "text": "$i\\hbar\\frac{d\\Psi}{dt}=-[V-\\frac{-\\hbar^2}{2m}\\nabla^2]\\Psi$"
+            },
+            "annotations": [
+                {
+                    "text": "$(top,left)$",
+                    "showarrow": false,
+                    "xref": "paper",
+                    "yref": "paper",
+                    "xanchor": "left",
+                    "yanchor": "top",
+                    "x": 0,
+                    "y": 1,
+                    "textangle": 10,
+                    "bordercolor": "#0c0",
+                    "borderpad": 3,
+                    "bgcolor": "#dfd",
+                },
+                {
+                    "text": "$(right,bottom)$",
+                    "xref": "paper",
+                    "yref": "paper",
+                    "xanchor": "right",
+                    "yanchor": "bottom",
+                    "x": 0.2,
+                    "y": 0.7,
+                    "ax": -20,
+                    "ay": -20,
+                    "textangle": -30,
+                    "bordercolor": "#0c0",
+                    "borderpad": 3,
+                    "bgcolor": "#dfd",
+                    "opacity": 0.5,
+                },
+                {"text": "$not-visible$", "visible": false},
+                {
+                    "text": "$^{29}Si$",
+                    "x": 0.7,
+                    "y": 0.7,
+                    "showarrow": false,
+                    "xanchor": "right",
+                    "yanchor": "top",
+                },
+                {
+                    "text": "$^{17}O$",
+                    "x": 0.7,
+                    "y": 0.7,
+                    "ax": 15,
+                    "ay": -15,
+                    "xanchor": "left",
+                    "yanchor": "bottom",
+                },
+            ],
+        },
+    }
+
+    app.layout = html.Div(
+        children=[
+            dcc.Markdown("# h1 tag with inline MathJax: $E=mc^2$", mathjax=True),
+            dcc.Markdown(
+                """
+                    ## h2 tag with MathJax block:
+                    $$
+                    \\frac{1}{(\\sqrt{\\phi \\sqrt{5}}-\\phi) e^{\\frac25 \\pi}} =
+                    1+\\frac{e^{-2\\pi}} {1+\\frac{e^{-4\\pi}} {1+\\frac{e^{-6\\pi}}
+                    {1+\\frac{e^{-8\\pi}} {1+\\ldots} } } }
+                    $$
+                    ## Next line.
+                """,
+                mathjax=True,
+            ),
+            dcc.Graph(mathjax=True, id="graph-with-math", figure=fig),
+            dcc.Markdown("### No MathJax: Apple: $2, Orange: $3"),
+            dcc.Graph(id="graph-without-math", figure=fig),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.percy_snapshot("mkdw008 - markdown and graph with/without mathjax")
```

Comment:
This looks like the one case where we have both graph and markdown with math. Can we do something here to verify that only one copy of MathJax is requested? count the number of `<script>` elements pointing to MathJax?

Reply:
Ignore this comment, per the weird webpack behavior https://github.com/plotly/dash/pull/1685#issuecomment-877199466 as I [mentioned above](https://github.com/plotly/dash/pull/1949#discussion_r824278502)